scenario/state.py

```diff
@@ -444,6 +446,14 @@ def broken_event(self) -> "Event":
         )
 
 
+_DEFAULT_IP = "42.42.42.42"
+DEFAULT_JUJU_DATABAG = {
```

Comment:
I assume this are the actual keys that you get with Juju? I don't have the experience to know myself, and the only [doc[(https://github.com/juju/juju/blob/ecd609d9e8700e87f630b6fb8c8b6690f211092d/doc/charms-in-action.txt#L254) I could find looks quite outdated (it only has private-address). (And my local Juju is borked right now so having a look is tricky).